h2o-automl/src/main/java/ai/h2o/automl/AutoML.java

```diff
@@ -689,6 +689,10 @@ private void setCommonModelBuilderParams(Model.Parameters params) {
       }
       //TODO: add a check that gives an error when class_sampling_factors, max_after_balance_size is set and balance_classes = false
     }
+
+    //TODO will/should custom fold_column assignment be removed if we set keep_cross_validation_fold_assignment to false?
+    // Maybe we should because user do know assignments as it is he who is providing `fold_column` in the first place.
+    params._keep_cross_validation_fold_assignment = buildSpec.build_control.nfolds != 0 && buildSpec.build_control.keep_cross_validation_fold_assignment;
```

Comment:
Thoughts about this comment @ledell ?

Reply:
as we discussed with @ledell we will keep custom fold_column even if _keep_cross_validation_fold_assignment is set to false. Kind of special case.